package/MDAnalysis/coordinates/null.py

```diff
@@ -57,4 +57,9 @@ def __init__(self, filename, **kwargs):
         pass
 
     def _write_next_frame(self, obj):
+        try:
```

Comment:
I went ahead and added this check to the null writer. My understanding is that the null writer should behave like all other writers, so it should complain if we pass it a Timestep. Thoughts anyone?